src/pages/SignUp/SignUp.js

```diff
@@ -1,7 +1,59 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { KAKAO_AUTH_URL } from './Components/OAuth';
+import * as S from './SignUp.styled';
 
 const SignUp = props => {
-  return <>SignUp</>;
+  const signUpKakao = () => {
+    window.location.replace(KAKAO_AUTH_URL);
+  };
+  return (
+    <div className="signUpContainer">
+      <S.SignUpBox>
+        <S.SocialLoginContainer>
+          <S.TextContainer>
+            <S.Text fontSize={30} fontWeight={700}>
+              회원가입
+            </S.Text>
+            <S.Text>
+              와디즈 신규회원 가입하고 <br /> 다양한 혜택을 받아보세요
+            </S.Text>
+          </S.TextContainer>
+          <S.IconContainer>
+            <S.KakaoSignUpBox>
+              <S.SocialLoginImageBox onClick={signUpKakao}>
+                <S.SocialLoginImage
+                  src="./images/SignUpImages/kakao.png"
+                  alt="카카오"
+                />
+              </S.SocialLoginImageBox>
+              <S.SocialLoginImageBox>
+                <S.SocialLoginImage
+                  src="./images/SignUpImages/naver.png"
+                  alt="네이버"
+                />
+              </S.SocialLoginImageBox>
+              <S.SocialLoginImageBox>
+                <S.SocialLoginImage
+                  src="./images/SignUpImages/apple.png"
+                  alt="애플"
+                />
+              </S.SocialLoginImageBox>
+            </S.KakaoSignUpBox>
+            <S.IsAlreadyLogin>
+              이미 와디즈 계정이 있나요? <Link to="/sign-in">로그인</Link>
+            </S.IsAlreadyLogin>
+          </S.IconContainer>
+        </S.SocialLoginContainer>
+        <div className="imageBox">
+          <S.AdImage
+            src="./images\SignUpImages\wadizSignup.PNG"
```

Comment:
`/` 확인해주세요! 그리고 이미지의 경로는 public 폴더 하위의 절대경로로 접근하시면 됩니다.
```suggestion
            src="/images/SignUpImages/wadizSignup.PNG"
```

Reply:
상대경로로 전체 변경 하였습니다!